ui_trt.py

```diff
@@ -78,61 +66,93 @@ def on_ui_tabs():
-
-                        with FormRow(elem_classes="checkboxes-row", variant="compact"):
-                            use_fp16 = gr.Checkbox(label='Use half floats', value=True, elem_id="trt_fp16")
-
-                        button_export_trt = gr.Button(value="Convert ONNX to TensorRT", variant='primary', elem_id="trt_convert_from_onnx")
-                        button_show_trt_command = gr.Button(value="Show command for conversion", variant='secondary', elem_id="trt_convert_from_onnx")
+                    with gr.Tab(label="Convert ONNX to TensorRT (Preset configs)"):
+                        gr.HTML(value="<p style='margin-bottom: 0.7em'>Set configuration to true to build multiple TensorRT engines. (Warning build time increases per selected preset)</p>")
+                        
+                        # trt_source_filename_preset = gr.Textbox(label='Onnx model filename', value="", elem_id="trt_source_filename_preset")
+                        with gr.Column(variant="panel"):
+                            trt_source_filename_preset = gr.Dropdown(ONNX_MODELS, label="ONNX Model", info="Select an Onnx model to convert to TensorRT. Found in models/Unet-onnx")
+                            refresh_onnx_db = gr.Button("Refresh List.")
+                        trt_filename = gr.Textbox(label='Output filename', value="", elem_id="trt_filename", info="Leave empty to use the same name as onnx and put results into models/Unet-trt directory", visible=False)
+                        
+                        with gr.Row():
+                            
+                            with gr.Column(label="Presets 512x512"):
+                                profile_512_512_1 = gr.Checkbox(label='512x512x1', value=False, elem_id="profile_512_512_1")
+                                profile_512_512_2 = gr.Checkbox(label='512x512x2', value=False, elem_id="profile_512_512_2")
+                                profile_512_512_4 = gr.Checkbox(label='512x512x4', value=False, elem_id="profile_512_512_4")
+                                
+                            with gr.Column(label="Presets 768x768"):
+                                profile_768x768x1 = gr.Checkbox(label='768x768x1', value=False, elem_id="profile_768_768_1")
+                                profile_768x768x2 = gr.Checkbox(label='768x768x2', value=False, elem_id="profile_768_768_2")
+                                profile_768x768x4 = gr.Checkbox(label='768x768x4', value=False, elem_id="profile_768_768_4")
+
+                        use_fp16 = gr.Checkbox(label='Use half floats', value=True, elem_id="trt_fp16")
+                        button_export_preset_trt = gr.Button(value="Convert ONNX to TensorRT", variant='primary', elem_id="button_export_preset_trt")
+
+
+                    # with gr.Tab(label="Convert ONNX to TensorRT (Custom)"):

```

Comment:
remove all the commented code

Reply:
We might need this UI later in the future.